app/src/main/java/org/dosomething/letsdothis/tasks/CampaignDetailsTask.java

```diff
@@ -0,0 +1,44 @@
+package org.dosomething.letsdothis.tasks;
+import android.content.Context;
+
+import org.dosomething.letsdothis.data.Campaign;
+import org.dosomething.letsdothis.network.NetworkHelper;
+import org.dosomething.letsdothis.network.models.ResponseCampaign;
+
+import co.touchlab.android.threading.eventbus.EventBusExt;
+import co.touchlab.android.threading.tasks.Task;
+
+/**
+ * Created by izzyoji :) on 4/17/15.
+ */
+public class CampaignDetailsTask extends Task
```

Comment:
extend the baseNetworkErrorHandler task which deals with any weird network issues